packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts

```diff
@@ -401,6 +417,14 @@ export class IncomingRequestsController extends ConsumptionBaseController {
         }
     }
 
+    private async checkRelationshipStatus(peer: CoreAddress) {
+        const relationship = await this.relationshipResolver.getRelationshipToIdentity(peer);
+        // no relationship is for a template request on new relationship
+        if (relationship && relationship.status !== RelationshipStatus.Active) {
+            throw CoreErrors.requests.noMatchingRelationship(peer.toString());
```

Comment:
the error message is wrong. It says pending or active, but you only check for active here.